### ISSUE TITLE
Coalesce multiple blocks with the same definition?

### ISSUE THREAD

**Issue by kylepaulsen** [digisfera/useref#42]
#39 addresses the issue of having multiple blocks with the same build target, however the output html has multiple script tags that point to the same built script. Is there a reason for this? It seems like it should only create one script to replace the final build block parsed with the same build target, and all the other matching build blocks just get removed.

Also, instead of adding an index to the end of duplicate build targets, shouldn't node-useref just concat more script paths into the first "section" that is created for the build target?

If it's ok, I could submit a PR for this.

**Comment by manuelcabral** [comment]
Why do you need this feature instead of using a single block? I think it would be not obvious to have this as the default behaviour.

**Comment by kylepaulsen** [comment]
My case is a strange one, and I ended up writing my own build script to handle it. Maybe it's for the best.

In any case, I have <link rel="import">'s in my head that I want to be converted into a string and put into a js var. This code can then be concatenated with all my other js code. The problem is that I have all my js just before the body end tag. I suppose I could have put it with the link rel's... But like I said, I wrote my own build script anyway. I would still be willing to work on this though if anyone wanted.

It's still strange that someone's PR ( #39 ) got merged allowing multiple blocks in the first place. I'm pretty sure having multiple <script> tags with the same JS resource being spit out in the output file is useless.

**Comment by ibolmo** [comment]
Yes, I'm running through this issue as well: https://github.com/jonkemp/useref/commit/a8905a4023c786486eee59488cb6b53680c4384e#commitcomment-20255614
